client/src/components/application/Transactions.vue

```diff
@@ -185,6 +185,14 @@
 
       await this.$store.dispatch('getTransactionsHistory', this.nr)
       this.$store.commit('setPendingTransactionsRequest', false)
+
+      // once page is mounted, set height of list for proper scrolling
+      this.$nextTick(() => {
+        const page = this.$el
+        const header = this.$el.querySelector('#transaction-header')
+        const list = this.$el.querySelector('#transaction-list-wrapper')
+        list.style.height = (page.clientHeight - header.clientHeight) + 'px'
+      })
```

Comment:
This is needed because we don't know, in the CSS, how high the header is (ie, depending on its data).